libraries/AP_GPS/AP_GPS.h

```diff
@@ -157,9 +174,8 @@ class AP_GPS
         return status(primary_instance);
     }
 
-    // Query the highest status this GPS supports
+    // Query the highest status this GPS supports (always reports GPS_OK_FIX_3D for the blended GPS)
```

Comment:
This "highest_supported_status" is only called for the real GPSs and it's only used in order to decide whether we should send RTK info to the GCS.  I think we don't want to send RTK data for the virtual blended GPS in any case so I've just added the warning and within the function protected against a dangerous access to unallocated memory.